Fix interaction with participant menu

danxuliu opened the pull request:
The participant menu is now closed when clicking anywhere once shown; before it was kept shown when clicking on the participant or when performing an action through an item in the menu itself, which was a strange behaviour for a menu.

Moreover, while waiting for the server response after clicking on an item of the menu now the menu toggle is replaced by a working icon; the button is shown again when the operation finishes, either successfully or with a failure. This lets the user know that the operation is still in progress while also preventing her from doing any further operation on that participant while there is one in progress.

nickvergessen commented:
> ![bildschirmfoto von 2018-09-28 10-23-27](https://user-images.githubusercontent.com/213943/46196942-ae866c80-c308-11e8-9c66-669348a1afb0.png)

The loading icon seems a bit of position? Or is that intended? (Using FF62)
Adding the `icon` class to the loading span fixes the position, but then it doesn't disappear anymore.

danxuliu commented:
>  The loading icon seems a bit of position? Or is that intended? (Using FF62)

No, not intended at all.

>  Adding the icon class to the loading span fixes the position, but then it doesn't disappear anymore.

The problem is that the icon is displayed as `inline` instead of as `block`; I do not get why, it was working as expected when I tested it, but now it does not... I will take a look to see what is going on.

danxuliu commented:
Fixed; it seems that as the icon was a `span` and as `hide()` was not called before `show()` (it has the `.hidden` CSS class from start) the default `display` value was set by jQuery, which in case of spans is `inline`.

Instead of changing to a `div` or something like that I replaced the calls to `show()` and `hide()` with `removeClass('hidden')` and `addClass('hidden')`, because at some point we will probably have to remove the calls to `show()` and `hide()` due to CSP preventing inline styles, so better start now ;-)

nickvergessen commented on js/views/participantlistview.js:
```diff
@@ -171,11 +176,19 @@
 			toggleMenuClass: function() {
 				this.ui.menu.toggleClass('open', this.menuShown);
 			},
+			closeMenu: function(e) {
```
ESlint error:
> 179:24 error 'e' is defined but never used no-unused-vars
